test/specs/scenarios/crossDevice.js

```diff
@@ -213,6 +216,31 @@ export const crossDeviceScenarios = async (lang) => {
         crossDeviceSubmit.clickOnSubmitVerificationButton()
         verificationComplete.verifyUIElements(copy)
       })
+
+      const isSubmitVerificationButtonDisabled = async() => {
+        try {
+          crossDeviceSubmit.submitVerificationButton.click()
```

Comment:
![image](https://user-images.githubusercontent.com/23279455/68761660-745f5380-060c-11ea-9e45-2f39707d7876.png)
Is there a way we could actually query and verify this `disabled` flag is there?

Reply:
Selenium can check if an element `isEnabled` so could add that as the first check in addition of the click.

Reply:
I was more thinking about sth like this: https://stackoverflow.com/a/47178516 
Just throwing an idea. Not sure this is actually possible.

Reply:
I originally used the `isEnabled` check, but switched to using a click event instead since that would better simulate what we're testing for. Though remembering what that does, it is the same as the click check also. Probably making the test actions clearer would be better.

Reply:
Yeah beacasue we say `isSubmitVerificationButtonDisabled` but we are actually checking it by `click()`. Not very clear.

Reply:
I still wonder whether we could query for the `disabled` attribute as someone did here:
https://stackoverflow.com/a/14933533

Reply:
Only problem with relying on the `disabled` attribute for the test is that there are other ways a button could be disabled. The `try...catch` for the second click is a better check IMO.

Reply:
I was thinking to add it as an additional check.